newapi/ooniapi/citizenlab.py

```diff
@@ -282,12 +282,19 @@ def add(self, username, cc, new_entry, comment):
 
             self.set_state(username, "IN_PROGRESS")
 
-    def update(self, username, cc, old_entry, new_entry, comment):
+    def update(self, username: str, cc: str, old_entry: dict, new_entry: dict, comment):
+        ks = ("url", "category_code", "category_desc", "date_str", "user", "notes")
```

Comment:
The column is called `date_added` not `date_str`

Reply:
It's also called `category_description` instead of `category_desc`

Reply:
`user` should be `source`

Reply:
Renamed